libcontainer/rootfs_linux.go

```diff
@@ -834,6 +843,31 @@ func rootfsParentMountPrivate(rootfs string) error {
 	return nil
 }
 
+// Make parent mount shared if it was not shared
+func rootfsParentMountShared(rootfs string) error {
+	sharedMount := false
+
+	parentMount, optionalOpts, err := getParentMount(rootfs)
+	if err != nil {
+		return err
+	}
+
+	optsSplit := strings.Split(optionalOpts, " ")
+	for _, opt := range optsSplit {
+		if strings.HasPrefix(opt, "shared:") {
+			sharedMount = true
+			break
+		}
+	}
+
+	// Make parent mount SHARED if it was not shared.
+	if !sharedMount {
+		return unix.Mount("", parentMount, "", unix.MS_SHARED, "")
```

Comment:
We can use `mount()` which is a simple `unix.Mount` wrapper.

https://github.com/opencontainers/runc/blob/a5777e871682c3973b73f4ef6bf7961eb22768dc/libcontainer/mount_linux.go#L53-L57

Reply:
Done, thanks.